client/blocks/upload-image/index.jsx

```diff
@@ -231,10 +234,12 @@ class UploadImage extends Component {
 
 		const isEditingDefaultImage = defaultImage && selectedImage === defaultImage.URL;
 
-		const media = isEditingDefaultImage ? defaultImage : {
-			src: selectedImage,
-			file: selectedImageName
-		};
+		const media = isEditingDefaultImage
+			? defaultImage
+			: {
```

Comment:
For some reason, Prettier adds one additional tab here. I had to format it manually after running Prettier, otherwise eslint wouldn't let me commit. Any ideas @jsnajdr?